packages/frontend-api/src/Model/Parameter/ParameterWithValuesFactory.php

```diff
@@ -63,4 +63,27 @@ public function createMultipleForProduct(Product $product): array
 
         return $parametersWithValues;
     }
+
+    /**
+     * @param array $productData
+     * @return array
+     */
+    public function createParameterArrayFromProductArray(array $productData): array
```

Comment:
Not really sure about the method name. Should not it by in plural `createParametersArrayFromProductArray`?

Singular should be used to map single parameter which could be used for assigning into returning array. Maybe cosider renaming it and refactoring?

Reply:
okay, in 9be1288de